app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MyNewRailsApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+  </head>
+
+  <body>
+    <div id="root"></div>
```

Comment:
- Good job done so far, but according to the project requirements, this should be placed in the view file for the `root#index` controller i.e in the index.html.erb file for the static controller. Please fix this 👍 👍